src/main/java/uk/gov/hmcts/reform/civil/model/genapplication/GAPbaDetails.java

```diff
@@ -17,19 +17,19 @@
 public class GAPbaDetails {
 
     private final DynamicList applicantsPbaAccounts;
-    private final String pbaReference;
+    private final String serviceReqReference;
     private final Fee fee;
     private final PaymentDetails paymentDetails;
     private final LocalDateTime paymentSuccessfulDate;
 
     @JsonCreator
     GAPbaDetails(@JsonProperty("applicantsPbaAccounts") DynamicList applicantsPbaAccounts,
-                 @JsonProperty("pbaReference") String pbaReference,
+                 @JsonProperty("pbaReference") String serviceReqReference,
                  @JsonProperty("fee") Fee fee,
                  @JsonProperty("paymentDetails") PaymentDetails paymentDetails,
                  @JsonProperty("paymentSuccessfulDate") LocalDateTime paymentSuccessfulDate) {
         this.applicantsPbaAccounts = applicantsPbaAccounts;
-        this.pbaReference = pbaReference;
+        this.serviceReqReference = serviceReqReference;
```

Comment:
Also, I am not sure if we should wipe out the PBA reference, that we accept from the user via screen, with service reference. Any specific reason for this change?

Reply:
As discussed, Retained the value Pbarefrence and added a new field Service request Reference in GApbaDetails to store the service request response